components/FeatureHighlight/src/MDCFeatureHighlightViewController.m

```diff
@@ -192,6 +192,22 @@ - (void)setBodyColor:(UIColor *)bodyColor {
   self.view.bodyColor = bodyColor;
 }
 
+- (void)setTitleFont:(UIFont *)titleFont {
+  self.view.titleFont = titleFont;
```

Comment:
This will call loadView, which is likely unexpected.
Please maintain local ivars for all of these properties and see if view exists before setting.
Additionally, set these view.properties in viewDidLoad.